packages.config

```diff
@@ -21,6 +21,10 @@
   <package id="Microsoft.DiaSymReader" version="1.1.0" />
   <package id="System.ValueTuple" version="4.3.1" />
   <package id="Microsoft.VisualFSharp.Msbuild.15.0" version="1.0.1" />
+  <package id="Microsoft.Build" version="14.3.0" />
```

Comment:
@KevinRansom These are needed when building the MonoPackaging: for better or worse the fsharp/fsharp build must be able to build using only packages from nuget.org.  

The Microsoft.VisualFSharp.Msbuild.15.0 is a private package but I don't think we actually need rely on it even for the DevDiv build

If we could avoid using any private packages for the compiler/build tools themselves that would b much appreciated.